lib/serve/index.ts

```diff
@@ -46,6 +46,8 @@ export class LisaServeApplicationStack extends Stack {
     public readonly restApi: FastApiContainer;
     public readonly modelsPs: StringParameter;
     public readonly endpointUrl: StringParameter;
+    public readonly managementKeySecret: Secret;
+    public readonly managementKeySecretNameStringParameter: StringParameter;
```

Comment:
why do we need to add these as externally readable members? i'd REALLY like to avoid exposing these, and instead just contrive the values in code, like you already did. I don't see any external references to these values (which is good), but because of the dangers of adding cross-stack dependencies that can block cdk updates (especially with constantly rotating values), I would much prefer to not expose these at all and keep them internal to this stack. (please see this change as a reference for my aversion to cross-stack param references: https://github.com/awslabs/LISA/commit/e2ae9efa5e169354f24f7d81ffac8007c6599d52 )